pythonPackages.nix

```diff
@@ -7,4 +7,12 @@ in {
   pyscf = callPackage ./pyscf { };
   pyquante = callPackage ./pyquante { };
   pychemps2 = callPackage ./chemps2/PyChemMPS2.nix { };
+  qcelemental = callPackage ./qcelemental { };
+  qcengine = callPackage ./qcengine { };
+  gau2grid-1_3_1 = callPackage ./gau2grid/1_3_1.nix { };
+  gau2grid-2_0_4 = callPackage ./gau2grid/2_0_4.nix { };
+  gau2grid = callPackage ./gau2grid { };
+  pylibefp = callPackage ./pylibefp { };
+  psi4 = callPackage ./psi4 { };
+  psi4-9b60184 = callPackage ./psi4/9b60184.nix { };
```

Comment:
I assume this an unstable/master branch version of psi4? I would change the naming scheme to be in line with what nixpkgs recommends:
* name the attribute `psi4Unstable`.
* name the nix file `unstable.nix`
* `version` should be the date of the commit.

Reply:
It is a snapshot, from master, yes. Have adapted it.